docs/getstarted.md

```diff
@@ -17,83 +17,86 @@ This example creates a single cluster in a single VPC, then configures two route
 
 ![Single cluster/VPC service-to-service communications](images/example1.png)
 
-**Steps**
+### Steps
 
-   **Set up Service-to-Service communications**
+**Set up service-to-service communications**
 
-1. Create the Kubernetes Gateway `my-hotel`:
+1. Use aws cli to create a VPC Lattice service network, with the name `my-hotel`:
    ```bash
-   kubectl apply -f examples/my-hotel-gateway.yaml
-   ```
-   ***Note***: By default, the gateway (lattice service network) is not associated with cluster's VPC.  To associate a gateway (lattice service network) to VPC, `my-hotel-gateway.yaml` includes the following annotation.
-   ```
-   apiVersion: gateway.networking.k8s.io/v1beta1
-   kind: Gateway
-   metadata:
-     name: my-hotel
-     annotations:
-       application-networking.k8s.aws/lattice-vpc-association: "true"
+   aws vpc-lattice create-service-network --name my-hotel 
+   {
+    "arn": "<my-hotel-sn-arn>",
+    "authType": "NONE",
+    "id": "<my-hotel-sn-id>",
+    "name": "my-hotel"
+   }
    ```
-1. Verify that `my-hotel` gateway is created (this could take about five minutes):
+   
+1. Create the Lattice ServiceNetworkVpcAssociation between current k8s cluster VPC and `my-hotel` service network:
```

Comment:
nit: we should stay consistent with how we refer to VPC Lattice resources. Above, we say "VPC Lattice service network", but here we say "Lattice ServiceNetworkVpcAssociation".